src/controls/ArrayControl.tsx

```diff
@@ -58,6 +65,15 @@ export function ObjectArrayControl({
   const options: ArrayControlOptions =
     (uischema.options as ArrayControlOptions) ?? {}
 
+  // Note: For primative arrays, ArrayControlProps.data is an array
+  // For object arrays, ArrayLayoutProps.data is a number
+  const dataSource: any[] =
```

Comment:
Could you try unknown instrad of any here? Any is pretty dangerous and can typically be replaced by unknown without too much work